test/utils/specUtils.js

```diff
@@ -8,3 +8,51 @@ function eventFire(el, etype){
     el.dispatchEvent(evObj);
   }
 }
+
+angular.module('patternfly.testutils',['patternfly.utils'])
```

Comment:
Wondering about the placement of these here versus in the test itself.  Will these ever be used outside of the context of the transclude directive test?